docs/features/detection_mode.md

```diff
@@ -0,0 +1,29 @@
+# Detection Mode
```

Comment:
```suggestion
# Detection mode
```

Reply:
I would like to keep Page titles captitalized